### ISSUE TITLE
Please handle this case

### ISSUE THREAD

**Issue by shiv19** [TheOriginalJosh/nativescript-slides#109]
When I run my app with nativescript-slides on android
I get this in the logs

```
JS: clipToBounds with value false is not supported on Android. You can use this.android.getParent().setClipChildren(false) as an altern
ative
```

Can you handle this please by adding a check on line 502 of nativescript-slides.ts
```
if(app.android){
	this.android.getParent().setClipChildren(false);
}
if(app.ios){
	footerInnerWrap.clipToBounds = false;
}
```

**Comment by TheOriginalJosh** [comment]
Thank you for the heads up @shiv19  I'll look into this here

**Comment by TheOriginalJosh** [comment]
resolved in 2.2.12

**Comment by shiv19** [comment]
Thanks Josh 👍
